Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -3673,7 +3673,7 @@ - (void)_showSafeBrowsingWarningWithURL:(NSURL *)url title:(NSString *)title war
 - (void)_showWarningViewWithURL:(NSURL *)url title:(NSString *)title warning:(NSString *)warning detailsWithLinks:(NSAttributedString *)details completionHandler:(void(^)(BOOL, NSURL *))completionHandler
```

Comment:
I should've left this as `_showSafeBrowsingWarningViewWithURL` because it's only called from other `_showSafeBrowsingWarning` SPI.

Reply:
I actually think it's better as _showWarningViewWithURL because I hope it is used for more diverse warnings in the future.

Reply:
Yeah, we'll just need to add another parameter for the warning data in that case.